noir-projects/noir-contracts/contracts/app/token_contract/src/main.nr

```diff
@@ -510,6 +535,7 @@ pub contract Token {
     /// in the enqueued call).
     #[private]
     fn mint_to_private(
+        // TODO(benesjan): This allows minter to set arbitrary `from`. That seems undesirable. Will nuke it in a followup PR.
```

Comment:
Reviewer agrees?

Reply:
Yes, this is a problem. `from` here means the account that will share the tagging secret with `to`. In the future this might get removed entirely.